presto-main/src/main/java/com/facebook/presto/security/FileBasedSystemAccessControl.java

```diff
@@ -62,6 +68,7 @@ private FileBasedSystemAccessControl(List<CatalogAccessControlRule> catalogRules
             implements SystemAccessControlFactory
     {
         static final String CONFIG_FILE_NAME = "security.config-file";
+        static final String PERIOD = "security.refresh-period";
```

Comment:
This constant is already defined in `FileBasedAccessControlConfig.setRefreshPeriod`'s annotation. Would it be possible to reuse? Is `FileBasedAccessControlConfig` actually used to parse the config file?

Reply:
Would it be possible to create corresponding `FileBasedSystemAccessControlConfig` and inject it here via Guice?

Then `FileBasedSystemAccessControl.Factory` would need to be managed by Guice too, but that shouldn't be a problem

Reply:
Using guice here would affect in not that minor refactor of `com.facebook.presto.security.AccessControlManager`

Reply:
Hm, I think it would just be:
```
@Inject
public AccessControlManager(TransactionManager transactionManager, FileBasedSystemAccessControlFactory fileBasedSystemAccessControlFactory) {
   ...
   addSystemAccessControlFactory(fileBasedSystemAccessControlFactory);
}
```